lms/static/js/student_account/views/FormView.js

```diff
@@ -172,7 +172,7 @@
 
                         if (key) {
                             test = this.validate(elements[i]);
-                            if (test.isValid) {
+                            if (test.isValid || $el.attr('id') === 'login-email') {
```

Comment:
Ok I think I'm getting confused. Now that I check the code the one that I'm disabling the error is in the new login view. And the old view is the one that doesn't check?